examples/encode_video/encode.py

```diff
@@ -0,0 +1,63 @@
+from scannerpy import Database, DeviceType, Job
```

Comment:
Can you clean up this file a little bit if we're going to keep it in the examples directory?